src/ResourceManagement/Insights/Microsoft.Azure.Insights/Generated/Management/Insights/Models/LogProfileResource.cs

```diff
@@ -25,19 +25,26 @@ public partial class LogProfileResource : Resource
         /// </summary>
         /// <param name="location">Resource location</param>
+        /// <param name="storageAccountId">the resource id of the storage
+        /// account to which you would like to send the Activity Log.</param>
+        /// <param name="locations">List of regions for which Activity Log
+        /// events should be stored or streamed. It is a comma separated list
+        /// of valid ARM locations including the 'global' location.</param>
         /// <param name="id">Azure resource Id</param>
         /// <param name="name">Azure resource name</param>
         /// <param name="type">Azure resource type</param>
         /// <param name="tags">Resource tags</param>
-        /// <param name="storageAccountId">the resource id of the storage
-        /// account.</param>
-        /// <param name="serviceBusRuleId">the resource id of the service bus
-        /// rule.</param>
-        /// <param name="locations">the locations.</param>
-        /// <param name="categories"> the categories.</param>
-        /// <param name="retentionPolicy">the retention policy for this
-        /// log.</param>
-        public LogProfileResource(string location, string id = default(string), string name = default(string), string type = default(string), System.Collections.Generic.IDictionary<string, string> tags = default(System.Collections.Generic.IDictionary<string, string>), string storageAccountId = default(string), string serviceBusRuleId = default(string), System.Collections.Generic.IList<string> locations = default(System.Collections.Generic.IList<string>), System.Collections.Generic.IList<string> categories = default(System.Collections.Generic.IList<string>), RetentionPolicy retentionPolicy = default(RetentionPolicy))
+        /// <param name="serviceBusRuleId">The service bus rule ID of the
+        /// service bus namespace in which you would like to have Event Hubs
+        /// created for streaming the Activity Log. The rule ID is of the
+        /// format: '{service bus resource ID}/authorizationrules/{key
+        /// name}'.</param>
+        /// <param name="categories">the categories of the logs. These
+        /// categories are created as is convenient to the user. Some values
+        /// are: 'Write', 'Delete', and/or 'Action.'</param>
+        /// <param name="retentionPolicy">the retention policy for the events
+        /// in the log.</param>
+        public LogProfileResource(string location, string storageAccountId, System.Collections.Generic.IList<string> locations, string id = default(string), string name = default(string), string type = default(string), System.Collections.Generic.IDictionary<string, string> tags = default(System.Collections.Generic.IDictionary<string, string>), string serviceBusRuleId = default(string), System.Collections.Generic.IList<string> categories = default(System.Collections.Generic.IList<string>), RetentionPolicy retentionPolicy = default(RetentionPolicy))
```

Comment:
Same comment as above. 
This resource model contains all that the PUT operation requires, with the same structure after it is serialized.